v3/wrap/README.md

```diff
@@ -172,15 +172,15 @@ We used the Ethereum wrapper's `encodeFunction` method to encode calldata for Un
 // An abridged copy of src/router/utils.ts in the Uniswap v3 wrapper
 
 import {
```

Comment:
NIT: we should probably inject these code snippets using doc-snippets, or just point the reader to source files (src/ or examples/)

Reply:
i think this should somehow be fixed with the new wraps-doc approach...